employee-management/interface-document.md

```diff
@@ -82,23 +81,44 @@
 
 ##### Parameter error
 
-|HTTP Status|Error code|Error message|Description |
+|HTTP Status|Error code|Error message|Description|
 |:----|:----|:----|:----|
-|400|invalid_id|Employee ID is invalid|The requested Employee ID is contains Non-Nummeric characters. |
+|400|invalid_employee_id|Employee ID is invalid.|The requested Employee ID is contains Non-Nummeric characters.|
+|400|invalid_employee_id_lenght|Character too long.|Request Employee ID too long.|
 
 ##### Sample Error Response
 
-- When a single error is used
+- When returning a single error
 
 ```json
 {
-  "code": "invalid_id ",
+  "code": "invalid_employee_id ",
   "message": "Employee ID is invalid."
 }
 ```
 
+- When returning multiple errors at the same time
+
+```json
+{
+  "errors": [
+    {
+      "code": "invalid_employee_id",
+      "message": "Employee ID is invalid."
+    },
+    {
+      "code": "invalid_employee_id_lenght",
+      "message": "Character too long."
+    }
+  ]
+}
+```
+
 ## Search Employee API
 
+- Returns the employee information,which corresponding to the condition was described in the [Query paramenters](#query-parameters)
+- Returns the list of employee information which ascending by employee_id
```

Comment:
`ascending` là thể loại từ gì vậy

`employee_id` -> `Employee ID`

Reply:
ascending là động từ ạ. Em sửa lại: Sort the list of employee information result by ascending employee_id có đúng ko ạ.

Reply:
Em sẽ sửa lại Employee ID

Reply:
`Sort the list of employee information result by ascending`
câu thì đúng, nhưng ko phù hợp với ngữ cảnh của câu phía trên